homeassistant/components/image_processing/facebox.py

```diff
@@ -10,20 +10,26 @@
 import requests
 import voluptuous as vol
 
-from homeassistant.const import ATTR_NAME
+from homeassistant.const import (
+    ATTR_ENTITY_ID, ATTR_NAME)
 from homeassistant.core import split_entity_id
 import homeassistant.helpers.config_validation as cv
 from homeassistant.components.image_processing import (
     PLATFORM_SCHEMA, ImageProcessingFaceEntity, ATTR_CONFIDENCE, CONF_SOURCE,
-    CONF_ENTITY_ID, CONF_NAME)
+    CONF_ENTITY_ID, CONF_NAME, DOMAIN)
 from homeassistant.const import (CONF_IP_ADDRESS, CONF_PORT)
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_BOUNDING_BOX = 'bounding_box'
+ATTR_CLASSIFIER = 'classifier'
 ATTR_IMAGE_ID = 'image_id'
 ATTR_MATCHED = 'matched'
 CLASSIFIER = 'facebox'
+DATA_FACEBOX = 'facebox_classifiers'
+EVENT_CLASSIFIER_TEACH = 'image_processing.teach_classifier'
```

Comment:
Is this our standard format for event names? Have you checked other platforms or components that fire events?

Reply:
I don't see a consistent format, so took my lead from `EVENT_DETECT_FACE = 'image_processing.detect_face'`. I think `teach_classifier` is pretty a accurate description, but am open to  other suggestions?